recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
 {% set shortversion = "5.0" %}
```

Comment:
Was just noticing this wasn't updated. That said, it doesn't seem to be used anywhere. Is it ok to drop it?

Reply:
Yeah, I'd say so. Also, sorry for missing it.

Reply:
Thanks for confirming. Added PR ( https://github.com/conda-forge/llvmdev-feedstock/pull/33 ) to drop it.